wagtailio/settings/base.py

```diff
@@ -149,8 +149,8 @@
 # Basic auth settings
 if os.environ.get("BASIC_AUTH_ENABLED", "false").lower() == "true":
     MIDDLEWARE.insert(0, "baipw.middleware.BasicAuthIPWhitelistMiddleware")
-    BASIC_AUTH_LOGIN = "wagtailio"
-    BASIC_AUTH_PASSWORD = "showmewagtailio"
+    BASIC_AUTH_LOGIN = env.get("BASIC_AUTH_LOGIN")
+    BASIC_AUTH_PASSWORD = env.get("BASIC_AUTH_PASSWORD")
```

Comment:
```suggestion
    BASIC_AUTH_LOGIN = env.get("BASIC_AUTH_LOGIN", "wagtailorg")
    BASIC_AUTH_PASSWORD = env.get("BASIC_AUTH_PASSWORD", "showmewagtailorg")
```
it is good practice to have a sensible fallback